src/mod_vault.c

```diff
@@ -436,18 +444,20 @@ static int wolfVaultGetItemHeader(wolfVaultCtx* ctx, size_t itemPos)
 
 static int wolfVaultGetItemData(wolfVaultCtx* ctx, wolfVaultItem* item)
 {
-    int ret = 0;
+    int ret;
 
 #ifdef WOLFKM_VAULT_ENC
     /* make sure key has been setup */
     ret = wolfVaultSetupKey(ctx);
     if (ret != 0) {
         return ret;
     }
+#else
+    ret = 0;
 #endif
```

Comment:
don't need this #else as ret is initialized to zero above.

Reply:
Sorry...I should have said if you had initialized to zero you would not need this else clause.  Not sure why you changed it to this form. Can you enlighten me?